host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@8.0.1_nvcc_xlf.cmake

```diff
@@ -108,6 +108,8 @@ set(CUDA_SEPARABLE_COMPILATION ON CACHE BOOL "")
 
 set(AXOM_CUDA_ARCH "sm_70" CACHE PATH "")
 
+set(AXOM_USE_CUB ON CACHE BOOL "")
```

Comment:
Is there a case where Axom uses CUB independent of RAJA?

Reply:
No, in fact we use the CUB version that is bundled with RAJA. The reason that this option was added is b/c there was a linking issue with earlier versions of CUDA and we needed to turn it off.  If we are not using CUB, the code calls `std::stable_sort()` on the CPU, which would require data motion and currently require using UM for supplied bounding boxes. 

In the future, we would use the`RAJA::sort()` when that is available.

Please, let me know if you have any additional questions/suggestions on this.

Reply:
RAJA also allows external versions of CUB and camp (i.e., not the RAJA submodules). MARBL builds this way. We should also make sure that that works. It is described here:
https://raja.readthedocs.io/en/master/getting_started.html

Reply:
Since this is orthogonal to this PR, I'll add a Github issue. I think it's just a matter of specifying a different CUB dir.

- [ ] Add Github issue.

Reply:
> There was a linking issue with earlier versions of CUDA

@gzagaris -- is this still an issue, or has it been resolved?  

If it has been resolved in our currently tested versions of Axom, I suggest enabling ``AXOM_USE_CUB`` to ``ON`` by default when using ``cuda`` and ``raja``?

Reply:
If axom want to use our own ``cub``, it will need to be sent to ``raja`` via spack and added as ``CUB_DIR`` to our host-config.  @rhornung67 -- do you know if ``raja``'s spack is set up to support that? 

It's an important use case for using a device-enabled axom, so we should try to set up our CI plan to test this functionality.

Reply:
> > There was a linking issue with earlier versions of CUDA
> 
> @gzagaris -- is this still an issue, or has it been resolved?
> 
> If it has been resolved in our currently tested versions of Axom, I suggest enabling `AXOM_USE_CUB` to `ON` by default when using `cuda` and `raja`?

It is not an issue -- I have already enabled it in this PR.

Reply:
See issue #224